app/views/user_imports/form.html.erb

```diff
@@ -1,34 +1,5 @@
 <%= elmo_form_for(@user_import, html: {multipart: true}) do |f| %>
   <p class="instructions"><%= formatted_instructions %></p>
-
-  <div id="file-uploader">
-    <%= hidden_field_tag(:saved_upload_id, "", id: "saved_upload_id") %>
-
-    <div id="dropzone-file-uploader" class="dropzone no-thumb">
-      <div class="dz-message">
-        <span class="message"><%= t("common.file_drop_message") %>&nbsp;</span>
-      </div>
-    </div>
-    <div class="dz-error-msg"></div>
-  </div>
-  <div class="submit-buttons">
-    <%= f.submit(t("common.import"), class: "btn btn-primary") %>
-    <%= render("file_upload/upload_in_progress_button", single_file: true) %>
-  </div>
-
-<% end %>
-
-<%= javascript_doc_ready do %>
-  ELMO.fileUploaderManager = new ELMO.Views.FileUploaderManager({el: "#new_user_import"});
-  options = <%=json(
-    el: "#file-uploader",
-    zoneId: "#dropzone-file-uploader",
-    genericThumbPath: nil,
-    postPath: "#{user_imports_path}/upload",
-    previewTemplate: render("file_upload/dropzone_preview"),
-    paramName: "file",
-    acceptedFileFormats: ".xlsx, .csv",
-  )%>
-  options.listener = ELMO.fileUploaderManager;
-  new ELMO.Views.FileUploaderView(options);
+  <%= render("file_upload/operation_file_uploader", form: f, operation_type_prefix: "user_imports",
+                                                    post_path: user_imports_path) %>
```

Comment:
You could just pass `import_type: :user` as the symbol and then do `send("#{import_type}_imports_path")` to get the post path. And a similar thing with the form CSS selector.